js/app.js

```diff
@@ -2,147 +2,184 @@
-
-if (military === 'yes') {
-//console.log('Nope. I was in the Air Force');
-  alert('Nope. I was in the Air Force.');
-} else if (military === 'no') {
-  //console.log('That\'s right! I was in the Air Force.');
-  alert('That\'s right! I was in the Air Force.');
-  score++;
-} else {
-  //console.log('I see you\'re playing hardball. Let\'s move on.');
-  alert('I see you\'re playing hardball. Let\'s move on.');
-}
-
-//Question 3:
-let hometown = prompt('Am I from Western New York?').toLowerCase();
-
-if (hometown === 'yes') {
-  //console.log('That\'s right! From the southern shores of Lake Ontario.');
-  alert('That\'s right! From the southern shores of Lake Ontario.');
-  score++;
-} else if (hometown === 'no') {
-  //console.log('WHAT?! Do you even know me at all? Of course I am!');
-  alert('WHAT?! Do you even know me at all? Of course I am!');
-} else {
-  //console.log('Don\'t like to follow the rules? That\'s okay.');
-  alert('Don\'t like to follow the rules? That\'s okay.');
-}
-
-//Question 4:
-let peach = prompt('Did I play in peach orchards as a kid?').toLowerCase();
-
-if (peach === 'yes') {
-  //console.log('I sure did. The climate of the great lakes helps to enable fruit tree orchards in such a cold climate.');
-  alert('I sure did. The climate of the great lakes helps to enable fruit tree orchards in such a cold climate.');
-  score++;
-} else if (peach === 'no') {
-  //console.log('Uhm, I actually did. Awkward.');
-  alert('Uhm, I actually did. Awkward.');
-} else {
-  //console.log('At a loss for words, eh?');
-  alert('At a loss for words, eh?');
-}
-
-//Question 5:
-let daughters = prompt('Do I have three daughters?').toLowerCase();
-
-if (daughters === 'yes') {
-  //console.log('Not yet, but you never know in the future! The answer is two.');
-  alert('Not yet, but you never know in the future! The answer is two.');
-} else if (daughters === 'no') {
-  //console.log('You\'re a keen observer! That\'s right, I have two.');
-  alert('You\'re a keen observer! That\'s right, I have two.');
-  score++;
-} else {
-  //console.log('You are a rebel, ain\'t ya?');
-  alert('You are a rebel, ain\'t ya?');
-}
-
-//Question 6:
-//Where does swiss cheese rank on my Top 10 List of favorite cheeses? hint: 1-10
-
-//Define number of attempts
-let attempts = 4;
+function quiz(score){
 
-//Define the answer
-let swiss = 6;
-
-for (let i = 0; i < attempts; i++) {
-  let favoriteCheese = prompt('Where does swiss rank on my top 10 list of favorite cheeses? You have four attempts! Hint: 1-10');
-  if (favoriteCheese === '6') {
-    //console.log('That\'s correct!');
-    alert('That\'s correct! Maybe it should be higher?');
-    score++;
-    break;
-  }
-  if (favoriteCheese < swiss) {
-    //console.log(`Just a little low! You have ${attempts - i - 1} attempts remaining.`);
-    alert(`Just a little low! You have ${attempts - i - 1} attempts remaining.`);
+  //Question 1:
+  function startGame(score){
```

Comment:
You're defining this function inside of another function. Try to have function definitions on the top level. Afterwards, you can set your quiz definition to run startGame() on it's own and get the same results with much more readable code.

Reply:
My initial comment was incorrect, you will not get the same results, because functions defined in a function will not inherently exist outside of the function. If you were to run quiz() in the console while inspecting your page, the quiz would run again. If you were to run startGame(score) the console will tell you it's not defined.